roles/jenkins/templates/transfer-to-slaves-script

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# NOTE: each single whitespace and newline close to the template is important!
+# - the script might stop to request an interactive confirmation of the host signature
+# - it's expected to run this as user 'jenkins' as it will have the right keys setup 
+
+transfer() {
+   rsync --exclude ".ssh/" --exclude "workspace/" --exclude "repository/" --exclude ".gem/" --exclude ".gradle/" --delete-during --archive --compress --progress --human-readable --one-file-system --stats ~/* $DESTINATION:~
```

Comment:
I would rather include specifically what is to be transferred. Too easy otherwise to catch something we don't want.